django_forest/utils/schema/validations.py

```diff
@@ -0,0 +1,74 @@
+from django.core.validators import MaxLengthValidator, MinLengthValidator, MaxValueValidator, MinValueValidator, \
+    RegexValidator
+
+
+VALIDATORS = {
+    'max_length': {
+        'type': 'is shorter than',
+        'message': 'Ensure this value has at most %(limit_value)d characters'
+    },
+    'min_length': {
+        'type': 'is longer than',
+        'message': 'Ensure this value has at least %(limit_value)d characters'
+    },
+    'max_value': {
+        'type': 'is less than',
+        'message': 'Ensure this value is less than or equal to %(limit_value)s characters'
+    },
+    'min_value': {
+        'type': 'is greater than',
+        'message': 'Ensure this value is greater than or equal to %(limit_value)s characters'
+    },
+}
+
+
+def handle_validator(validator, f):
+    if isinstance(validator, MaxLengthValidator) \
+            or isinstance(validator, MinLengthValidator) \
+            or isinstance(validator, MaxValueValidator) \
+            or isinstance(validator, MinValueValidator):
+        v = VALIDATORS[validator.code]
+        f['validations'].append({
+            'type': v['type'],
+            'message': v['message'] % {'limit_value': validator.limit_value},
+            'value': validator.limit_value
+        })
+    elif isinstance(validator, RegexValidator):
```

Comment:
If change above is accepted, maybe check against code here to be consistent.

Reply:
Hum, I've just experimented in regex case, the validator has no code. This is not really true, in fact, the code is `invalid` which is not something I'd like to check against...
Here is the definition for more insight:
```python
@deconstructible
class RegexValidator:
    regex = ''
    message = _('Enter a valid value.')
    code = 'invalid'
    inverse_match = False
    flags = 0
```
I think keeping the `isinstance` check is the best for now